ocrd/ocrd/cli/workspace.py

```diff
@@ -159,6 +167,54 @@ def workspace_add_file(ctx, file_grp, file_id, mimetype, page_id, force, local_f
     )
     workspace.save_mets()
 
+# ----------------------------------------------------------------------
+# ocrd workspace add-group
+# ----------------------------------------------------------------------
+
+@workspace_cli.command('add-group')
+@click.option('-m', '--mimetype', help="Media type of the file", required=True)
+@click.option('-f', '--force', help="Continue even if mets:file same ID already exists", default=False, is_flag=True)
+@click.argument('file_grp_dir', type=click.Path(dir_okay=True, file_okay=False, readable=True, resolve_path=True), required=True)
```

Comment:
What if the directory contains other kinds of files (e.g. thumbnails for images, or backup copies, or a mix of different file types)? Wouldn't it be a better interface to call with a glob pattern which the shell expands to a list of files?

Reply:
I fear that users will fail to quote glob calls properly. But if I'm arguing for regexes, I can't argue against globs.

Reply:
I think I actually meant globbing by the shell, so you could also just pass a space-delimited list of path names.

IMO we have 2 principle routes:
- pattern for both file names and page IDs controlled by regex argument
- shell globbing or other pathname expansion, and guesswork/heuristics on our side

Reply:
files can be passed directly or as a glob expression expanded by python's `glob` module.

the pattern to match files against is passed as `--regex`. I'll post an example forthwith.